### ISSUE TITLE
An addon to adjust color tags on forum posts so that they are accessible

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6356]
### Why this would be helpful

Some of the colors used in forum posts aren't ideal for accessibility. This can happen because:
- The post author doesn't realize that it's a problem
- You have customized the colors of Scratch (like using a dark background), causing an otherwise accessible color to become inaccessible (just think about how hard it would be to see dark gray on a black background)

### How this addon works

It would be nice if there was an addon that could adjust those inaccessible colors so that they followed a set of guidelines, making sure the user can read them.

### Additional context

This is similar to the setting `terminal.integrated.minimumContrastRatio` from VS Code.

**Comment by Secret-chest** [comment]
It could have three modes: Darken (for light mode, with percentage), Lighten (for dark mode, with percentage) and Invert.

**Comment by mybearworld** [comment]
This could also be an opportunity to make the colors in code tags work with a dark bacckground.

**Comment by mxmou** [comment]
Related: #2325

**Comment by mxmou** [comment]
> This could also be an opportunity to make the colors in code tags work with a dark bacckground.

Dark mode already changes those colors, but the contrast of some of them could still be improved.

**Comment by DNin01** [comment]
> It could have three modes: Darken (for light mode), Lighten (for dark mode) and Invert, along with a minimum contrast ratio treshold (when the colour is below that treshold, it starts lightening/darkening until it satisfies that treshold

I think this could be done automatically by comparing the color to the background set in `dark-www`.
